sceptre/cli/init.py

```diff
@@ -48,9 +49,9 @@ def init_project(ctx, project_name):
     Creates PROJECT_NAME project folder and a config.yaml with any
     required properties.
     """
-    cwd = os.getcwd()
-    sceptre_folders = {"config", "templates"}
-    project_folder = os.path.join(cwd, project_name)
+    context = SceptreContext(project_path=ctx.obj["project_path"])
+    sceptre_folders = {context.config_path, context.templates_path}
```

Comment:
Should this be using `SceptreContext(..).full_templates_path` instead?

Reply:
Not in this case. `sceptre_folders` is later used in line 66 to iterate over.